src/App/Color.h

```diff
@@ -41,9 +41,9 @@ class AppExport Color
 public:
     /**
      * Defines the color as (R,G,B,A) whereas all values are in the range [0,1].
-     * \a A defines the transparency.
+     * @param A defines the transparency.
      */
-    explicit Color(float R=0.0,float G=0.0, float B=0.0, float A=0.0);
+    explicit Color(float R=0.0,float G=0.0, float B=0.0, float A=1.0);
```

Comment:
One should evaluate this change very carefully!!

Issue is, that some functions App::Color considers `A = 1.0` as transparent while other functions considers `A = 0.0` as transparent.

I have here chosen, to redefine transparent as `A = 0` as it harmonizes how alpha values are handled elsewhere like QColor and HTML. Though this could break various stuff! Some code might need to be adopted to this change, as it will else cause objects to suddenly become transparent.

This is exactly what happened to me, as I tried implementing alpha value in the `asValue` function: suddenly all objects became transparent, as I had implemented the functionality according to documentation where transparency is defined as `A = 1`, while `asPackedRGBA` defined transparency as `A = 0` making all alpha values suddenly being retrieved opposite than saved.